src/jwtf/src/jwtf.erl

```diff
@@ -292,6 +307,24 @@ hmac_verify(Algorithm, Message, HMAC, SecretKey) ->
             throw({bad_request, <<"Bad HMAC">>})
     end.
 
+jose_to_der(Signature) ->
+    NumLen = 8 * byte_size(Signature) div 2,
+    <<R:NumLen, S:NumLen>> = Signature,
+    SigValue = #'ECDSA-Sig-Value'{r = R, s = S},
+    public_key:der_encode('ECDSA-Sig-Value', SigValue).
+
+der_to_jose(Alg, Signature) ->
+    #'ECDSA-Sig-Value'{r = R, s = S} = public_key:der_decode('ECDSA-Sig-Value', Signature),
+    Len = rs_len(Alg),
+    <<R:Len, S:Len>>.
+
+rs_len(<<"ES256">>) ->
+    256;
+rs_len(<<"ES384">>) ->
+    384;
+rs_len(<<"ES512">>) ->
+    512.
```

Comment:
Is there a reason to prefer this implementation over e.g.
```erlang
rs_len(<<"ES", SizeBin/binary>>) ->
    binary_to_integer(SizeBin).

```

Reply:
I like your version better.